### DISCUSSION TITLE
Question for the MedusaJS team about frontend starter

### DISCUSSION THREAD

**Discussion by ensidialabs** [medusajs/medusa#4533] [Q&A] [answered]
What's the difference between nextjs-starter-medusa and vercel-commerce (medusa-edition)? They develop in parallel. And which of them is better and more promising to use? which will be better supported.

❤ MedusaJS

**Comment by srindom** [comment, marked as answer]
Thanks for the question! 

The Vercel commerce template implements Medusa-specific connectors to make Medusa the backend for the template. This template's features and capabilities are defined by what the Vercel team adds.

Our starter template ([nextjs-starter-medusa](https://github.com/medusajs/nextjs-starter-medusa)), is built by us and therefore implements more of Medusa's features. For example, the template includes a pre-built checkout flow, customer accounts, and pluggable search, all of which are not available in the Vercel commerce template without further customization. 

Both templates are demonstrations of how Nextjs, Medusa, and Vercel make a great stack. We will generally include new Medusa features in nextjs-starter-medusa when they become available, but we can't always do so with the Vercel commerce template as we don't define this template's features.
